test-suite/Makefile

```diff
@@ -241,7 +241,7 @@ $(foreach S,$(VSUBSYSTEMS),$(eval $(call vdeps,$(S))))
 # Summary
 #######################################################################
 
-summary_dir = echo $(1); find $(2) -name '*.log' -print0 | xargs -0 tail -q -n1 | sort
+summary_dir = echo $(1); find $(2) -name '*.log' -print0 | xargs -0 -L 999  tail -q -n1 | sort
```

Comment:
```suggestion
# some versions of tail do not accept more than ~1k arguments
summary_dir = echo $(1); find $(2) -name '*.log' -print0 | xargs -0 -L 999  tail -q -n1 | sort
```

Reply:
OK, pushed (and extended).

Reply:
I see no push

Reply:
I had beforehand forgotten to commit. Now ok.